shared/search/tokens.ts

```diff
@@ -53,6 +53,10 @@ export function findSearchToken({
         }
     }
 
+    if (start >= end) {
```

Comment:
Isn't `start === end` a legal representation of an empty range? (Or do we not handle empty ranges?)

Reply:
Empty ranges (as in `[x, x)`) can't be hovered over, so that's fine!

I'd have to check to see if this a half-open interval, though. Either way: it was the previous behavior, and we're missing at most single-letter basic code intel, which isn't going to give us much precision even if we perform it.